velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -462,6 +462,18 @@ class PlanBuilder {
       const std::vector<std::string>& aggregationInputs,
       std::string groupIdName = "group_id");
 
+  /// Add an ExpandNode using specified projections. See comments for
+  /// ExpandNode class for description of this plan node.
+  ///
+  /// @param projections A list of projection expressions. Each value in one
+  /// projection is the column name, null or the const value.
+  ///
+  /// For example,
+  ///
+  ///     .expand({"a", "b", "", "1"})
```

Comment:
Are you using "" for null? How do you decide what type that null is and what name to use for it in the output? Let's use "null:<type> AS <name>" instead. Similarly, for non-null constants let's use "1 as <name>".